src/Meta/Alert_Meta.php

```diff
@@ -207,6 +222,21 @@ private function get_rules_group(): ACF\Field_Group {
 		return $group;
 	}
 
+	private function get_colors_field(): ACF\Field {
+		$field = new ACF\Field( self::NAME . '_' . self::FIELD_COLOR );
+
+		$field->set_attributes( [
+			'label'         => __( 'Backgroung Color', 'tribe-alerts' ),
```

Comment:
```suggestion
			'label'         => __( 'Background Color', 'tribe-alerts' ),
```
This is misspelled, but I think we should be calling this "Color Theme" anyways, and perhaps provide an `instructions` key for the field to explain a bit more about what it does.